doc/api/errors.md

```diff
@@ -1004,6 +1004,22 @@ Used when a callback is called more then once.
 can either be fulfilled or rejected but not both at the same time. The latter
 would be possible by calling a callback more then once.
 
+<a id="ERR_NET_INVALID_LISTEN_ARG"></a>
+### ERR_NET_INVALID_LISTEN_ARG
```

Comment:
@Trott @refack The reason that I used this new error was that the listen arg had many valid combinations of types/schema. I was not sure about the way that I handled it. I see your point and I agree with you. I will replaced it, with `ERR_INVALID_ARG_TYPE`.